src/OpenLoco/Title.cpp

```diff
@@ -62,7 +63,7 @@ namespace OpenLoco::Title
         CompanyManager::updatingCompanyId(CompanyManager::getControllingId());
         if (isPaused())
         {
-            togglePause(true);
+            GameCommands::do_20();
```

Comment:
Is this safe to do? Will the game command actually be executed?

Reply:
Our old `togglePause` was incomplete compared to what the game command does. However, what I'm doing here is still different, as the original routine called `0x00431E32` (the togglePause game command) directly. Any thoughts?

Reply:
I'm just slightly concerned that during loading/saving the game command runner might be in a state that disallows the toggle pause command. I don't know enough about the game command runner to be sure this isn't an issue.

Reply:
Given the `togglePause` game command is rather short to begin with, let's just implement it to hopefully solve this potential problem.